go/internal/ocxcli/doctor_command.go

```
@@ -0,0 +1,343 @@
+	if deps.Catalog == nil {
+		deps.Catalog = CollectDoctorCatalogStateDefault
+	}
+	if deps.Hints == nil {
+		deps.Hints = CollectDoctorHints
+	}
+	return deps
+}
+
+var doctorCommandTODOs = []DoctorTODOSection{}
+
+func doctorTODO(section DoctorTODOSection) []string {
```

golangci-lint (2.13.2)

[error] func doctorTODO is unused

(unused)